src/pages/UserPage/UserPage.tsx

```diff
@@ -130,13 +130,38 @@ const UserPage: FC = () => {
       name: 'notify_user',
       label: t('label.user_activation_notification'),
       ariaLabel: t('label.user_activation_notification'),
-      rules: {
-        required: true,
-      },
       className: classes.checkboxInputClass,
     },
   ]
 
+  const onArhiveSubmit: SubmitHandler<FormValues> = useCallback(
+    async (values) => {
+      const payload: UserStatusType = { ...values, institution_user_id }
+
+      navigate('/settings/users')
+
+      try {
+        await archiveUser(payload)
+
+        showNotification({
+          type: NotificationTypes.Success,
+          title: t('notification.announcement'),
+          content: t('success.user_archived', { name }),
+        })
+      } catch (errorData) {
+        const typedErrorData = errorData as ValidationError
```

Comment:
**[blocker]**

No need to handle validation errors we get from archiving endpoint just to show the errors in deactivate/activate modal. Can cause some very weird behaviour where we get errors from archive user endpoint because we are passing fields there that it doesnt accept and set them as errors for the "activate" modal form. When the user then opens the "activate/deactivate" user modal, they will see some errors that don't make any sense + archiving can potentially fail because of these extra parameters that we send